lessons/Early-Math/distributive-property/langs/en-us/index.adoc

```diff
@@ -40,121 +40,264 @@ Students develop an understanding of the @vocab{Distributive Property} by examin
+}
+
+@lesson-instruction{
+Turn to @printable-exercise{pages/mental-math.adoc} and look at the first problem.
+}
+
+Our goal here is to make the math easier by creating an equivalent expression that we can solve in our heads! So, instead of finding the product of 70 and 39, we are going to multiply 70 by the difference of 40 and 1.
+
+@slidebreak
 
 @lesson-instruction{
-- Turn to @printable-exercise{pages/mental-math.adoc} and look at the first problem.
-- Our goal here is to make the math easier by creating an equivalent expression that we can solve in our heads! So, instead of finding the product of 70 and 39, we are going to multiply 70 by the difference of 40 and 1.
 - Complete the next Circle of Evaluation, which shows that we are going to _distribute_ 70.
-- We can now compute our solution by finding difference between two products.
-- Do you find this process more efficient than the standard algorithm for multiplication? Explain.
-** _Note: Without ample practice, there is a good chance that students will not find this process efficient. Like any solving strategy, it takes practice!_
+- Observe how we can now compute our solution by finding the difference between two products.
+}
+
+@QandA{
+@Q{Do you find this process more efficient than the standard algorithm for multiplication? Explain.}
+}
+
+@lesson-instruction{
 - Complete the rest of the page by creating equivalent expressions that we can solve in our heads.
 }
 
+
+@teacher{Without ample practice, there is a good chance that students will not find this process efficient. Like any solving strategy, it takes practice!}
+
 === Synthesize
 
 - How can you multiply two 2-digit numbers using mental math?
-- Can you think of a multiplication problem that would be _easier_ to solve using the Distributive Property?
-** _Note: We recommend inviting many students to share the problems they come up with. Make a list on the board for students to review together._
 - What sorts of problems are simpler to compute using the Distributive Property?
+- Can you think of a multiplication problem that would be _easier_ to solve using the Distributive Property?
+
+
+@teacher{We recommend inviting many students to share the problems they come up with. Make a list on the board for students to review together.}
+
+== Programming Exploration: Distribution @duration{20 minutes}
+
+=== Overview
+
+Extending concepts explored earlier in the lesson, students consider whether various functions that we use when coding are distributive.
+
+=== Launch
+
+We've learned a lot about how the distributive property works. We know that multiplication can distribute over addition or subtraction, allowing us to complete otherwise complex computations with efficiency and ease.
+
+@slidebreak
+
+But what about our programming functions? For instance, does `scale` distribute over `beside`?!
```

Comment:
There is no such thing as a "programming function" in Pyret or WeScheme. They are simply **functions**. I suggest rewriting this as "But what about functions that deal with images?"

The more you make "programming" sound different from math, the more ammunition you give math teachers to resist. I can just hear someone saying "maybe that's cool for programming functions, but this is a _math class_. Why should I make time for something that isn't math?"

Reply:
heard. This makes good sense.